apps/pie-docs/package.json

```diff
@@ -39,6 +39,7 @@
     "markdown-it-attrs": "4.1.6"
   },
   "dependencies": {
+    "@justeat/f-cookie-banner": "4.7.0",
```

Comment:
I did have a question about whether this would work as a devDependency, but I'm not sure if it makes a difference